qe/io.py

```diff
@@ -52,13 +62,15 @@ def load_integrals(
         f = bz2.open(fcidump_path)
     else:
         f = open(fcidump_path)
+        used_zip = False
 
     # Only non-zero integrals are stored in the fci_dump.
     # Hence we use a defaultdict to handle the sparsity
     n_orb = int(next(f).split()[2])
 
-    for _ in range(3):
-        next(f)
+    # Using takewhile we 'take lines from the file' while '/' has not been found
+    # Needed so we can start reading data on the integrals.
+    lines = list(takewhile(lambda line: "/" not in manipulate_line(line, used_zip), f))
```

Comment:
Needed to be equal to a list for some reason that I do not know.

Reply:
It's the difference between an `iterator`  and a `list`.  Maybe `https://stackoverflow.com/questions/25653996/what-is-the-difference-between-list-and-iterator-in-python` can help.